### DISCUSSION TITLE
SwayWM very unstable

### DISCUSSION THREAD

**Discussion by Foxelyss** [Manjaro-Sway/manjaro-sway#467] [Q&A] [answered]
It happens ONLY on Manjaro Sway. If install SwayWM manually on, for example, Manjaro XFCE it works perfectly, but SwayWM on Manjaro Sway crashes a lot. Sway suddenly crashes with different messages followed by "Broken Pipe". It sometimes happens right on logging in, sometimes after 15 minutes,but sometimes on window transparency effect when focusing on window......
Graphics cards : Intel HD Graphics 530, nVidia GeForce GT710 (Nouveau)
[log.txt](https://github.com/manjaro-sway/manjaro-sway/files/10976168/log.txt)

**Comment by Foxelyss** [comment]
To remove these errors I just needed to change this variables: WLR_RENDERER_ALLOW_SOFTWARE WLR_DRM_NO_MODIFIERS, from one to zero.

**Reply by boredland** [reply]
We should drop that one or find a dynamic way to set/not set that based on if software rendering/drm is available.

**Comment by boredland** [comment, marked as answer]
We're now trying to feature-detect the hardware acceleration a bit. Hope this works for you?
